docs/testing-operators.md

```diff
@@ -1,120 +1,404 @@
+```
+git clone https://github.com/operator-framework/operator-marketplace.git
+git clone https://github.com/operator-framework/operator-courier.git
+git clone https://github.com/operator-framework/operator-lifecycle-manager.git
+```
+
+Before you begin your current working dir should look like the following, with `my-operator` as an example for the name of your bundle:
 
-`test-operator` injects a scorecard proxy container and volume into an operators' CSV manifest before deployment; this is necessary to get API server logs, from which the scorecard determines runtime test results. These modifications are not persistent, as they're only needed for testing.
+```
+my-operator
+operator-marketplace
+operator-courier
+operator-lifecycle-manager
+```
 
-**Note**: no explicit number of points or percentage is necessary to achieve before merging _yet_. These are suggestions to improve your operator.
+### Tools
 
 #### operator-courier
 
-The [`operator-courier verify`][courier] command verifies that a set of files is valid and can be bundled and pushed to [quay.io][quay]. Read the [docs][courier-docs] for more information.
+`operator-courier` is used for metadata syntax checking and validation. This can be installed directly from `pip`:
+
+```
+pip3 install operator-courier
+```
+
+#### Quay Login
+
+In order to test the Operator installation flow, store your Operator bundle on [quay.io](https://quay.io). You can easily create an account and use the free tier (public repositories only). To upload your Operator to quay.io a token is needed. This only needs to be done once and can be saved locally. The `operator-courier` repository has a script to retrieve the token:
+
+```
+./operator-courier/scripts/get-quay-token
+
+Username: johndoe
+Password: 
+{"token": "basic abcdefghijkl=="}
+```
+
+A token takes the following form and should be saved in an environment variable:
+
+```
+export QUAY_TOKEN="basic abcdefghijkl=="
+```
+### Linting
+
+`operator-courier` will verify the fields included in the Operator metadata (CSV). The fields can also be manually reviewed according to [the operator CSV documentation](https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md).
+
+The following command will run `operator-courier` against the bundle directory `my-operator/` from the example above.
+
+```
+operator-courier verify --ui_validate_io my-operator/
+```
+
+If there is no output, the bundle passed `operator-courier` validation. If there are errors, your bundle will not work. If there are warning we still encourage you to fix them before proceeding to the next step.
+
+### Push to quay.io
+
+The Operator metadata in it's bundle format will be uploaded into your namespace in [quay.io](http://quay.io).
+
+> The value for `PACKAGE_NAME` *must* be the same as in the operator's `*package.yaml` file and the operator bundle directory name. Assuming it is `my-operator`, this can be found by running `cat my-operator/*.package.yaml`.
```

Comment:
Why is this quoted? Question applies to all places. If you want to call out something, I suggest making it bold.